src/main/java/core/basesyntax/Zoo.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import core.basesyntax.type.Animals;
+import core.basesyntax.type.Birds;
+import core.basesyntax.type.Fishes;
+import core.basesyntax.type.Swimming;
+
+import java.util.List;
+
+public class Zoo<I> {
+    private I inhabitant;
+
+    public Zoo(I inhabitant) {
+        this.inhabitant = inhabitant;
+    }
+
+    public void add(I inhabitant) {
```

Comment:
try to simplify this logic
for example:
we can create method `getFishes` which will return the list of fishes, and then we will add fish there

Reply:
fixed